baseplate/events/queue.py

```diff
@@ -21,8 +21,28 @@
 import time
 import uuid
 
+from enum import Enum
+
 from . import MAX_EVENT_SIZE, MAX_QUEUE_SIZE
 from baseplate.message_queue import MessageQueue, TimedOutError
+from baseplate._utils import warn_deprecated
+
+
+class FieldKind(Enum):
+    """Enum of field kinds."""
+    NORMAL = None
```

Comment:
looks ok to me, am i looking at the wrong thing?

<img width="724" alt="screen shot 2016-06-20 at 2 39 51 pm" src="https://cloud.githubusercontent.com/assets/823747/16211208/0ef528e2-36f5-11e6-9b06-a619ee1ae5ea.png">